bin/agendash-standalone.js

```diff
@@ -11,23 +11,29 @@ program
   .option('-c, --collection <collection>', '[optional] Mongo collection, same as Agenda collection name, default agendaJobs', 'agendaJobs')
   .option('-p, --port <port>', '[optional] Server port, default 3000', (n, d) => Number(n) || d, 3000)
   .option('-t, --title <title>', '[optional] Page title, default Agendash', 'Agendash')
+  .option('-p, --path <path>', '[optional] Path to bind Agendash to, default /', '/')
   .parse(process.argv);
 
 if (!program.db) {
   console.error('--db required');
   process.exit(1);
 }
 
+if (!program.path.startsWith('/')) {
```

Comment:
If `path` is optional, it might be null/undefined in which case this would throw, no? Haven't tested tho, suppose it depends on how `commander` works. :-)

This might help though:
```
program?.path?.startsWith('/')
```

Reply:
Yes, it is optional, but it has the default value `"/"`.
I've tested without `path` CLI arg. It works.